Text/index.jsx

```diff
@@ -22,7 +23,7 @@ const Text = ({ children, size, weight, color }) => {
 
 Text.propTypes = {
   children: PropTypes.node,
-  size: PropTypes.oneOf(['extra-small', 'large', 'small']),
+  size: PropTypes.oneOf(['extra-small', 'large', 'mini', 'small']),
```

Comment:
Thanks for the comment, @hharnisc! `mini` fits in between `small` and regular. We could update our naming here and have `mini` be `small`, `small` be `extra-small`, and `extra-small` become `extra-extra-small`. 😀